src/main/java/dinkplugin/notifiers/DiaryNotifier.java

```diff
@@ -51,9 +53,9 @@ public void onTick() {
 
             if (initDelayTicks <= 0)
                 this.initCompleted();
-        } else if (diaryCompletionById.isEmpty() && isEnabled()) {
+        } else if (diaryCompletionById.isEmpty() && super.isEnabled()) {
             // mark diary completions to be initialized later
-            this.initDelayTicks = 8;
+            this.initDelayTicks = 4;
```

Comment:
I tested this change multiple times and I had no notifiers firing on log in or on world-hopping